pkg/k8sapi/k8sutils.go

```diff
@@ -85,15 +88,25 @@ func CheckAPIServerConnectivity() error {
 	if err != nil {
 		return err
 	}
-	clientSet, _ := kubernetes.NewForConfig(restCfg)
-
-	log.Infof("Testing communication with server")
-	version, err := clientSet.Discovery().ServerVersion()
+	restCfg.Timeout = 1 * time.Second
```

Comment:
can we change this to be 5 second to be safe?
with some slow response cluster, 1 second might be too aggressive.

change to 5 second shouldn't impact the this optimization much.

Reply:
How about 2 second timeout? because if a API server is taking more than 2 seconds to respond to version query its too slow and we want to know why and fix the issue.

Reply:
apiserver will easily take more than 2 seconds when scaling 10,000 pods, 600 nodes

Reply:
Version call shouldn't be impacted by the load in the server. 
If it's happening we will root cause, try and fix it, feel free to cut a support ticket and we can help debug.

Reply:
ok, didn't read the actual code, just shouting random comments 👍